superset/views/utils.py

```diff
@@ -351,11 +351,11 @@ def build_extra_filters(  # pylint: disable=too-many-locals,too-many-nested-bloc
     for filter_id, columns in default_filters.items():
         filter_slice = db.session.query(Slice).filter_by(id=filter_id).one_or_none()
 
-        filter_configs = (
-            json.loads(filter_slice.params or "{}").get("filter_configs") or []
-            if filter_slice
-            else []
-        )
+        filter_configs: List[Dict[str, Any]] = []
```

Comment:
Somehow `mypy` + `pre-commit` were picking up on this, unclear why as i didn't touch the file. Added type annotations here to move forward...